### PR TITLE
Enhancement/highestcpuusage

### LINKED ISSUE #769: When the system current limit occurs, the current CPU usage value cannot be obtained when sending a custom alarm.
## Issue Description

Type: feature request

When the system current limit occurs, the current CPU usage value cannot be obtained when sending a custom alarm.

### Describe what happened (or what feature you want)

When the system current limit occurs, the current CPU usage value cannot be obtained when sending a custom alarm.

### Describe what you expected to happen

When the system current limit occurs, send a custom alarm to get the current CPU usage value.

### How to reproduce it (as minimally and precisely as possible)

1. The getHighest CpuUsage method is added to the SystemRuleManager class
2. 
3. 

### Tell us your environment
1.6.0

### PR DESCRIPTION
**Pull request by yikangfeng**
### Describe what this PR does / why we need it
Open getHighest CpuUsage to send custom alarm when system current limit occurs。

### Does this pull request fix one issue?

Fixes #769

### Describe how you did it

Adding getHighestCpuUsage Method

### Describe how to verify it

Access via SystemRuleManager. getHighestCpuUsage

### Special notes for reviews

### CODE REVIEW

File: sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRuleManager.java [hunk 1]
```diff
@@ -241,6 +241,14 @@ public static double getHighestSystemLoad() {
     public static void setHighestSystemLoad(double highestSystemLoad) {
         SystemRuleManager.highestSystemLoad = highestSystemLoad;
     }
+    
+    public static double getHighestCpuUsage() {
+        return highestCpuUsage;
+    }
+
+    public static void setHighestCpuUsage(double highestCpuUsage) {
```
Comment (sczyh30) [inline sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRuleManager.java]: Maybe the setter method is not needed.

**Comment by sczyh30** [comment]
And I think if you're going to retrieve the current CPU usage and load of your system, you may get them via:

```java
SystemRuleManager.getCurrentCpuUsage();
SystemRuleManager.getCurrentSystemAvgLoad()
```

The `highestCpuUsage` is only the value of the `SystemRule`.

File: sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRuleManager.java [hunk 2]
```diff
@@ -241,6 +241,10 @@ public static double getHighestSystemLoad() {
     public static void setHighestSystemLoad(double highestSystemLoad) {
         SystemRuleManager.highestSystemLoad = highestSystemLoad;
     }
+    
+    public static double getHighestCpuUsage() {
```
Comment (sczyh30) [inline sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRuleManager.java]: Maybe it's better to be `getCpuUsageThreshold()`?

Comment (yikangfeng) [reply sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/system/SystemRuleManager.java]: Well

**Comment by sczyh30** [comment]
Thanks for contributing!

**Pull request merged**